lib/ansible/modules/cloud/google/drive_append_to_sheet.py

```diff
@@ -0,0 +1,203 @@
+      A Google account and the Python client library for Google APIs are requirements.
+      Also you must have the Google Sheets API activated and valid credentials for this handy.
+      The necessary API steps can be done here:
+      https://developers.google.com/sheets/api/quickstart/python#step_1_turn_on_the"
+
+options:
+    columns:
+        description:
+            - This is the variable that should be appended to the GSheet
+        required: true
+        aliases: [ 'name' ]
+    sheetID:
+        description:
+            - This is the Sheet ID
+        required: true
+    client_id:
+        description:
+            - This is the Google OAuth client_id
+              It ends with .apps.googleusercontent.com
+        required: true
+    client_secret:
+        description:
+            - This is the Google OAuth client_secret
+        required: true
+    sheetName:
+        description:
+            - This is the Sheet inside of the GSheet that we will add information
+        required: false
+    range:
+        description:
+            - This is the range
+        default: 'A1:A1'
+        required: false
+
+requirements:
+    - "google-api-python-client"
+    - "google-auth-httplib2"
+    - "google-auth-oauthlib"
+
+author:
+    - Chris Blum (@mulbc)
+'''
+
+EXAMPLES = '''
+# Append one row with three columns
+- name: Write stuff to GSheet
+  drive_append_to_sheet:
+    name: ["column A", "column B", "column C"]
+    sheetID: 1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms
+    client_id: teststestst.apps.googleusercontent.com
+    client_secret: S3CR3T
+
+# Append one row with three variables
+- name: Write stuff to GSheet
+  drive_append_to_sheet:
+    columns: ["{{ ansible_fqdn }}", "{{ ansible_kernel }}", "{{ ansible_lsb.release }}"]
+    sheetID: 1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms
+    client_id: teststestst.apps.googleusercontent.com
+    client_secret: S3CR3T
+'''
+
+RETURN = '''
+new_columns:
+    description: The columns that were appended
+    returned: success
+    type: list
+message:
+    description: The response we got from the Google API
+    returned: success
+    type: dict
+'''
+try:
+    import pickle
+    import os.path
+    from googleapiclient.discovery import build
+    from google_auth_oauthlib.flow import InstalledAppFlow
+    from google.auth.transport.requests import Request
+    import_ok = True
+except ImportError:
+    import_ok = False
+
+from ansible.module_utils.basic import AnsibleModule
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
+
+
+def main():
+    """Shows basic usage of the Sheets API.
+    Prints values from a sample spreadsheet.
+    """
+
+    # define available arguments/parameters a user can pass to the module
+    module_args = dict(
+        columns=dict(type='list', required=True, aliases=['name']),
+        sheetID=dict(required=True),
```

Comment:
Please add types to every parameter in the argspec.
So if you did not add it, it most likely should be: `type='str'`.

Reply:
resolved by fb49811